website/docs/r/cloud_aws_integrations.html.markdown

```diff
@@ -80,223 +82,223 @@ resource "newrelic_cloud_aws_integrations" "bar" {
     tag_value                = "tag value"
   }
   elasticache {
     aws_regions              = ["us-east-1"]
     fetch_tags               = true
-    metrics_polling_interval = 6000
+    metrics_polling_interval = 300
     tag_key                  = "tag key"
     tag_value                = "tag value"
   }
   api_gateway {
-    metrics_polling_interval = 6000
+    metrics_polling_interval = 300
     aws_regions              = ["us-east-1"]
     stage_prefixes           = ["stage prefix"]
     tag_key                  = "tag key"
     tag_value                = "tag value"
   }
   auto_scaling {
     aws_regions              = ["us-east-1"]
-    metrics_polling_interval = 6000
+    metrics_polling_interval = 300
   }
   aws_app_sync {
     aws_regions              = ["us-east-1"]
-    metrics_polling_interval = 6000
+    metrics_polling_interval = 300
   }
   aws_athena {
     aws_regions              = ["us-east-1"]
-    metrics_polling_interval = 6000
+    metrics_polling_interval = 300
   }
   aws_cognito {
     aws_regions              = ["us-east-1"]
-    metrics_polling_interval = 6000
+    metrics_polling_interval = 300
   }
   aws_connect {
     aws_regions              = ["us-east-1"]
-    metrics_polling_interval = 6000
+    metrics_polling_interval = 300
   }
   aws_direct_connect {
     aws_regions              = ["us-east-1"]
-    metrics_polling_interval = 6000
+    metrics_polling_interval = 300
   }
   aws_fsx {
     aws_regions              = ["us-east-1"]
-    metrics_polling_interval = 6000
+    metrics_polling_interval = 300
   }
   aws_glue {
     aws_regions              = ["us-east-1"]
-    metrics_polling_interval = 6000
+    metrics_polling_interval = 300
   }
   aws_kinesis_analytics {
     aws_regions              = ["us-east-1"]
-    metrics_polling_interval = 6000
+    metrics_polling_interval = 300
   }
   aws_media_convert {
     aws_regions              = ["us-east-1"]
-    metrics_polling_interval = 6000
+    metrics_polling_interval = 300
   }
   aws_media_package_vod {
     aws_regions              = ["us-east-1"]
-    metrics_polling_interval = 6000
+    metrics_polling_interval = 300
   }
   aws_mq {
     aws_regions              = ["us-east-1"]
-    metrics_polling_interval = 6000
+    metrics_polling_interval = 300
   }
   aws_msk {
     aws_regions              = ["us-east-1"]
-    metrics_polling_interval = 6000
+    metrics_polling_interval = 300
   }
   aws_neptune {
     aws_regions              = ["us-east-1"]
-    metrics_polling_interval = 6000
+    metrics_polling_interval = 300
   }
   aws_qldb {
     aws_regions              = ["us-east-1"]
-    metrics_polling_interval = 6000
+    metrics_polling_interval = 300
   }
   aws_route53resolver {
     aws_regions              = ["us-east-1"]
-    metrics_polling_interval = 6000
+    metrics_polling_interval = 300
   }
   aws_states {
     aws_regions              = ["us-east-1"]
     metrics_polling_interval = 6000
```

Comment:
This might need a correction to a value that is not 6000

Reply:
@abasha1234 just to be clear: please change the `metrics_polling_interval` to a value that is not 6000 (since 6000 is invalid). This must be fitting in some category (of 300 or 900 or 3600 seconds of its minimum polling interval), so please identify that from the below list and change 6000 to the right value accordingly.

Reply:
updated the metrics_polling_interval for `aws_states` to 300